extensions/amp-access/0.1/login-dialog.js

```diff
@@ -267,7 +267,7 @@ export class WebLoginDialog {
     }
     dev().fine(TAG, 'Login done: ', result, opt_error);
     if (opt_error) {
-      this.reject_(opt_error);
+      this.reject_(user().createError('login failed', opt_error));
```

Comment:
Chances are that these are unmarked errors (the two above), and they are definitely user errors and not dev errors. How about making it so `Log#createError` doesn't change the user/dev mark if an error passed in already has one?